src/Infrastructure/Service/Logger/EventLoggerInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service\Logger;
+
+use SprykerSdk\Sdk\Core\Domain\Event\EventInterface;
+
+interface EventLoggerInterface
```

Comment:
This interface breaks layer dependency direction. A lot of services from the Application layer depend on it. Please, move it to the Application layer.